definitions/npm/react-css-modules_v3.x.x/flow_v0.53.x-/react-css-modules_v3.x.x.js

```diff
@@ -2,12 +2,13 @@
 import React from "react";
 
 declare module "react-css-modules" {
+  declare type StatelessComponent<P, S> = (props: P, state: S) => ?React$Element<any>;
```

Comment:
Should this not be a maybe type? `... => React$Element<any>`

Reply:
Yes, thanks.